D/difmap/build_tarballs.jl

```diff
@@ -0,0 +1,48 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "difmap"
+version = v"2.5.5"
```

Comment:
Can you add a comment here, explaining that `2.5e` has been mapped as `2.5.5`, so the next person who comes along understands?